src/js/lib/append-child.js

```diff
@@ -0,0 +1,11 @@
+export default function appendChild (parent, element) {
+
+	try {
+		parent.appendChild(element);
+	}
+	catch (err) {
+		console.warn(err);
+		document.body.insertBefore(element, document.body.firstChild);
```

Comment:
I dont think we want this here, as `appendChild` is used to add things like the close button, if that errored, we would have a close button added to the body.